portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -36,12 +38,12 @@
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     Query query = new Query("Comments").addSort("timestamp", SortDirection.DESCENDING);
+    query.addSort("username");
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
     PreparedQuery results = datastore.prepare(query);
-   
     List<String> commentsStored = new ArrayList<String>();
     for (Entity entity : results.asIterable()) {
-      String comment = (String) entity.getProperty("comment");
+      String comment = (String) entity.getProperty("username") + " : " + (String) entity.getProperty("comment");
```

Comment:
I think you would have an easier time if you returned an array of JSON objects instead of putting the `username` right in the comment string. You would need to make a `Comment` class like (pseudo-code)

```
class Comment:
  String username;
  String text;
```

And then the returned JSON from `doGet` would look like

```json
[{"username": "test", "text": "hello"}, {"username": "foo", "text": "world"}]
```